frameworks/helloworld/tests/test_sanity.py

```diff
@@ -32,12 +32,14 @@ def configure_package(configure_security):
     try:
         sdk_install.uninstall(FOLDERED_SERVICE_NAME, package_name=PACKAGE_NAME)
 
+        service_config = {"service": {"name": FOLDERED_SERVICE_NAME, "user": "root"}}
```

Comment:
For the record, this PR already makes the change to `1.8` at https://github.com/mesosphere/dcos-commons/pull/1398/files#diff-163288c917fefa95b80dcc812d66d234R3